media/img/hubs/cta-arrow.svg

```diff
@@ -1 +1,7 @@
-<svg width="14" height="12" viewBox="0 0 14 12" xmlns="http://www.w3.org/2000/svg"><g fill="none"><g fill="#9C432A"><polygon transform="translate(-1164 -1226)translate(1171 1232.024689)rotate(-90)translate(-1171 -1232.024689)" points="1169.7 1225 1169.7 1234 1166.8 1231.2 1165 1233 1169.7 1237.7 1169.7 1237.8 1169.8 1237.8 1171 1239 1172.2 1237.8 1172.3 1237.8 1172.3 1237.7 1177 1233 1175.2 1231.2 1172.3 1234 1172.3 1225"/></g></g></svg>
+<svg id="cta-arrow" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 14 24">
+  <style>
```

Comment:
Nit: could we lose the inline styles here by using `fill` properties directly on the paths?